sct.py

```diff
@@ -694,8 +694,8 @@ def list_repos(dist_type, dist_version):
               help='Scylla repo')
 @click.option('-d', '--linux-distro', type=str, help='Linux Distribution type')
 @click.option('-o', '--only-print-versions', type=bool, default=False, required=False, help='')
-@cloud_provider_option(default="aws", required=False, help="Cloud provided to query. Defaults to aws.")
-def get_scylla_base_versions(scylla_version, scylla_repo, linux_distro, only_print_versions, cloud_provider):  # pylint: disable=too-many-locals
+@click.option('-b', '--backend', type=click.Choice(SCTConfiguration.available_backends), help="Backend to use")
```

Comment:
so, right of now, the only "backend" we do something different is `azure`, because its first support being `5.2`... all others will behave exactly the same...
the idea here is that, once we will support this upgrade for k8s, we will need the different flow inside of this very same function, so it is already in place, for future use of k8s, but being used now, for Azure